programs/client/Client.cpp

```diff
@@ -538,6 +549,18 @@ class Client : public Poco::Util::Application
     {
         UseSSL use_ssl;
 
+        MainThreadStatus::getInstance();
+
+        /// Limit on total memory usage
+        size_t max_client_memory_usage = context->getSettings().max_memory_usage;
```

Comment:
memory tracker is set to zero at the beginning. This means that it is unlimited.